gnd/src/main/res/menu/nav_drawer_menu.xml

```diff
@@ -35,6 +35,10 @@
   <group
     android:id="@+id/group_other"
     android:checkableBehavior="none">
+    <item
```

Comment:
Please add a comment referencing a GitHub issue (`TODO(#...):`) to remove this menu item once the actual flow is implemented.

Reply:
Done!